tasks/debian.yaml

```diff
@@ -0,0 +1,30 @@
+# Always run including in check mode
+- name: selinux | check exists
+  stat:
+    path: /usr/sbin/getenforce
+  register: selinux_getenforce_st
+  check_mode: false
+
+- name: selinux | check enabled
+  become: true
+  command: /usr/sbin/getenforce
+  register: selinux_getenforce
+  check_mode: false
+  changed_when: false
+  when: selinux_getenforce_st.stat.exists
+
+- name: system packages | set selinux variable
+  set_fact:
+    selinux_enabled: >-
+      {{ selinux_getenforce_st.stat.exists and
+         selinux_getenforce.stdout != 'Disabled' }}
+
+- name: system packages | install selinux utilities
+  become: true
+  package:
+    name:
+      - libselinux-python
+      - libsemanage-python
+      - policycoreutils-python
```

Comment:
Just thinking that in the future, `Debian` may move to a `python3` default. Might make sense to move to `python3` everywhere, unless you think that is too much work